From 1ea86a149213b8fd1e3606bb68694a80eccaa5d1 Mon Sep 17 00:00:00 2001
From: Jakob Ruckel <jakob@ruckel.de>
Date: Thu, 21 Jan 2021 21:44:48 +0100
Subject: [PATCH] Make arc-dark darker

---
 PKGBUILD | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/PKGBUILD b/PKGBUILD
index fa54b6f..3299006 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -32,6 +32,27 @@ pkgver() {
 build() {
   cd "${_pkgname}"
 
+  SUBS=(
+    "s/#404552/#2e3341/gi"
+    "s/#d3dae3/#d9dfe7/gi"
+    "s/#383c4a/#272a38/gi"
+    "s/#d3dae3/#d9dfe7/gi"
+    "s/#4b5162/#393f53/gi"
+    "s/#3e4350/#2c313e/gi"
+    "s/#7c818c/#737a88/gi"
+    "s/#404552/#2e3341/gi"
+    "s/#353945/#242833/gi"
+    "s/#5294e2/#4192e6/gi"
+    "s/#383c4a/#272a38/gi"
+    "s/#444a58/#363b49/gi"
+    "s/#505666/#414757/gi"
+  )
+
+  for s in ${SUBS[@]}; do
+      find . -type f -exec sed -i "$s" {} \;
+  done
+
+
   meson --prefix=/usr build \
     -Dcinnamon_version="${_cinnamonver}" \
     -Dgnome_shell_version="${_gnomeshellver}" \
-- 
2.30.0

